gcc fails on RHEL6

```
2014/08/08 17:19:46 - INFO: [package:run_job] # @multilib_flags@ is still needed because this may use
2014/08/08 17:19:46 - INFO: [package:run_job] # /local/certik/tmp/gcc-ygygrgi2xtn3/host-x86_64-unknown-linux-gnu/gcc/xgcc -B/local/certik/tmp/gcc-ygygrgi2xtn3/host-x86_64-unknown-linux-gnu/gcc/ -B/local/certik/bld/gcc/ygygrgi2xtn3/x86_64-unknown-linux-gnu/bin/ -B/local/certik/bld/gcc/ygygrgi2xtn3/x86_64-unknown-linux-gnu/lib/ -isystem /local/certik/bld/gcc/ygygrgi2xtn3/x86_64-unknown-linux-gnu/include -isystem /local/certik/bld/gcc/ygygrgi2xtn3/x86_64-unknown-linux-gnu/sys-include    and -O2  -g -O2 -DIN_GCC    -W -Wall -Wwrite-strings -Wcast-qual -Wno-format -Wstrict-prototypes -Wmissing-prototypes -Wold-style-definition  -isystem ./include   -fpic -mlong-double-80 -g -DIN_LIBGCC2 -fbuilding-libgcc -fno-stack-protector  directly.
2014/08/08 17:19:46 - INFO: [package:run_job] # @multilib_dir@ is not really necessary, but sometimes it has
2014/08/08 17:19:46 - INFO: [package:run_job] # more uses than just a directory name.
2014/08/08 17:19:46 - INFO: [package:run_job] /bin/sh ../.././libgcc/../mkinstalldirs .
2014/08/08 17:19:46 - INFO: [package:run_job] /local/certik/tmp/gcc-ygygrgi2xtn3/host-x86_64-unknown-linux-gnu/gcc/xgcc -B/local/certik/tmp/gcc-ygygrgi2xtn3/host-x86_64-unknown-linux-gnu/gcc/ -B/local/certik/bld/gcc/ygygrgi2xtn3/x86_64-unknown-linux-gnu/bin/ -B/local/certik/bld/gcc/ygygrgi2xtn3/x86_64-unknown-linux-gnu/lib/ -isystem /local/certik/bld/gcc/ygygrgi2xtn3/x86_64-unknown-linux-gnu/include -isystem /local/certik/bld/gcc/ygygrgi2xtn3/x86_64-unknown-linux-gnu/sys-include    -O2  -g -O2 -DIN_GCC    -W -Wall -Wwrite-strings -Wcast-qual -Wno-format -Wstrict-prototypes -Wmissing-prototypes -Wold-style-definition  -isystem ./include   -fpic -mlong-double-80 -g -DIN_LIBGCC2 -fbuilding-libgcc -fno-stack-protector  -shared -nodefaultlibs -Wl,--soname=libgcc_s.so.1 -Wl,--version-script=libgcc.map -o ./libgcc_s.so.1.tmp -g -O2 -B./ _muldi3_s.o _negdi2_s.o _lshrdi3_s.o _ashldi3_s.o _ashrdi3_s.o _cmpdi2_s.o _ucmpdi2_s.o _clear_cache_s.o _trampoline_s.o __main_s.o _absvsi2_s.o _absvdi2_s.o _addvsi3_s.o _addvdi3_s.o _subvsi3_s.o _subvdi3_s.o _mulvsi3_s.o _mulvdi3_s.o _negvsi2_s.o _negvdi2_s.o _ctors_s.o _ffssi2_s.o _ffsdi2_s.o _clz_s.o _clzsi2_s.o _clzdi2_s.o _ctzsi2_s.o _ctzdi2_s.o _popcount_tab_s.o _popcountsi2_s.o _popcountdi2_s.o _paritysi2_s.o _paritydi2_s.o _powisf2_s.o _powidf2_s.o _powixf2_s.o _mulsc3_s.o _muldc3_s.o _mulxc3_s.o _divsc3_s.o _divdc3_s.o _divxc3_s.o _bswapsi2_s.o _bswapdi2_s.o _clrsbsi2_s.o _clrsbdi2_s.o _fixunssfsi_s.o _fixunsdfsi_s.o _fixunsxfsi_s.o _fixsfdi_s.o _fixdfdi_s.o _fixxfdi_s.o _fixunssfdi_s.o _fixunsdfdi_s.o _fixunsxfdi_s.o _floatdisf_s.o _floatdidf_s.o _floatdixf_s.o _floatundisf_s.o _floatundidf_s.o _floatundixf_s.o _divdi3_s.o _moddi3_s.o _udivdi3_s.o _umoddi3_s.o _udiv_w_sdiv_s.o _udivmoddi4_s.o cpuinfo_s.o sfp-exceptions_s.o addtf3_s.o divtf3_s.o multf3_s.o negtf2_s.o subtf3_s.o unordtf2_s.o fixtfsi_s.o fixunstfsi_s.o floatsitf_s.o floatunsitf_s.o fixtfdi_s.o fixunstfdi_s.o floatditf_s.o floatunditf_s.o fixtfti_s.o fixunstfti_s.o floattitf_s.o floatuntitf_s.o extendsftf2_s.o extenddftf2_s.o extendxftf2_s.o trunctfsf2_s.o trunctfdf2_s.o trunctfxf2_s.o getf2_s.o letf2_s.o eqtf2_s.o _divtc3_s.o _multc3_s.o _powitf2_s.o enable-execute-stack_s.o unwind-dw2_s.o unwind-dw2-fde-dip_s.o unwind-sjlj_s.o unwind-c_s.o emutls_s.o libgcc.a -lc && rm -f ./libgcc_s.so && if [ -f ./libgcc_s.so.1 ]; then mv -f ./libgcc_s.so.1 ./libgcc_s.so.1.backup; else true; fi && mv ./libgcc_s.so.1.tmp ./libgcc_s.so.1 && ln -s libgcc_s.so.1 ./libgcc_s.so
2014/08/08 17:19:46 - INFO: [package:run_job] /usr/bin/ld: crti.o: No such file: No such file or directory
2014/08/08 17:19:46 - INFO: [package:run_job] collect2: error: ld returned 1 exit status
2014/08/08 17:19:46 - INFO: [package:run_job] make[3]: *** [libgcc_s.so] Error 1
```

full log: https://gist.githubusercontent.com/certik/69d02274ecd88711c82b/raw/gistfile1.txt

This seems related to: http://stackoverflow.com/questions/6329887/compiling-problems-cannot-find-crt1-o

@jcftang any ideas how to fix this?

--- a/pkgs/gcc/gcc.yaml
+++ b/pkgs/gcc/gcc.yaml
@@ -18,7 +18,7 @@ build_stages:
     bash: |
       # Create symbolic links from lib64/* to lib/
       cd ${ARTIFACT}/lib
-      cp -s ../lib64/* .
+      ln -s ../lib64/* .
 
   # the checks will continue to run and not stop if there are errors,
   # this is here really for post build review.

